src/ant/src/AntennaChecker.tcl

```diff
@@ -58,3 +58,13 @@ proc check_antennas { args } {
   }
   return [ant::check_antennas $net_name $verbose]
 }
+
+
+
+sta::define_cmd_args "find_max_allowed_length" {net_name \
+                                                 layer}
+proc find_max_allowed_length { args } {
+  set net_name [lindex $args 0]
```

Comment:
Why is this a "command"? It seems like it is just a function that should be defined with arguments. Using "args" does not test the number of args passed or the type of the arguments. A command should take a layer name, not a layer number, which is ambiguous (routing layer? layer number? which is it?). Nothing checks that the layer is actually a number either.

Reply:
Would you recommend changing it a command and properly parse its argument or do I just make it a function with defined arguments? Regarding the 'layer' field, it represents the layer name so I will rename it to layer_name instead for clarity. Regarding checking, I am not making any checks here beause both 'net_name' and 'layer' are expected to be strings and checks are handled within the ANT module.

Reply:
What is the use model for this "command"? Is is going to be called as an API to the global router? Used in tests? I can't imagine it being very useful to a "user", which is why I don't like to see it pollute the command namespace and think it should just be a function in the ant namespace.

Reply:
@msaligane would you comment on the use model for this?  It is from the original UMich development.

Reply:
If it is not useful anymore, then please remove it. It was made initially for fastroute's antenna avoidance flow (~2y ago). I think we assigned numbers to the layers (I don't remember precisely but I think that's how layers were defined in ORFS).

Reply:
proc find_max_allowed_length { net_name layer_name}